### PR TITLE
Update installation notes

### PR DESCRIPTION
**Pull request by GliderGeek**
in the process of installation for development, i encounter some shortcomings in the documentation, which i try to rectify with this branch

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -84,8 +84,8 @@ Since the [XCSoar](http://www.xcsoar.org/) project already has much of the code
 implemented that is necessary for flight analysis, it makes sense to reuse that
 code where applicable. *SkyLines* is using XCSoar as a python library. This library
 is built and installed by the `xcsoar` python package. To build this library you
-might have to install additional libraries like `libcurl`, which can be installed
-on Debian/Ubuntu by executing `apt-get install libcurl4-openssl-dev`. Please have
+might have to install additional librarie `libcurl` and `freetype`, which can be installed
+on Debian/Ubuntu by executing `apt-get install libcurl4-openssl-dev` and `apt-get install libfreetype6-dev` respectively. Please have
```
Comment (Turbo87) [inline INSTALL.md]: you can use `apt-get install libcurl4-openssl-dev libfreetype6-dev` to install both dependencies with one command

File: INSTALL.md [hunk 2]
```diff
@@ -84,8 +84,8 @@ Since the [XCSoar](http://www.xcsoar.org/) project already has much of the code
 implemented that is necessary for flight analysis, it makes sense to reuse that
 code where applicable. *SkyLines* is using XCSoar as a python library. This library
 is built and installed by the `xcsoar` python package. To build this library you
-might have to install additional libraries like `libcurl`, which can be installed
-on Debian/Ubuntu by executing `apt-get install libcurl4-openssl-dev`. Please have
+might have to install additional librarie `libcurl` and `freetype`, which can be installed
```
Comment (Turbo87) [inline INSTALL.md]: can you keep the "libraries like" here?

File: INSTALL.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ Now you can install the python dependencies by calling:
     $ sudo pip install -e .
 
 *Note: You might have to install the additional Ubuntu/Debian packages
-`libpq-dev`, `python-dev` and `g++` for the `psycopg2` dependency.*
+`libpq-dev`, `python-dev` and `g++` for the `psycopg2` dependency. Furthermore the XCSoar tools might require additional dependecies (see XCSoar tools section)*
```
Comment (Turbo87) [inline INSTALL.md]: - "dependecies" is missing an "n"
- can you break the lines at 80 characters like the other are too?

**Comment by GliderGeek** [comment]
The missing faker module i encountered after running `./manage.py db create` .

This also gave another problem with gevent. Following [this SO post ](https://stackoverflow.com/questions/38602466/python-nameerror-name-protocol-sslv3-is-not-defined-when-using-gevent) i made the following change:

rename `PROTOCOL_SSLv3` to `PROTOCOL_SSLv23` inside `/usr/local/lib/python2.7/dist-packages/gevent/ssl.py`

File: INSTALL.md (hunk 3 above)
Comment (GliderGeek) [reply INSTALL.md]: will do, thanks

File: INSTALL.md (hunk 2 above)
Comment (GliderGeek) [reply INSTALL.md]: will do, thanks

File: INSTALL.md (hunk 1 above)
Comment (GliderGeek) [reply INSTALL.md]: will do, thanks

**Comment by GliderGeek** [comment]
currently i am running into problems installing the frontend.
this is the output i receive after executing `npm install` in the ember folder:
@Turbo87: any idea what happens?
```
$ npm install
npm WARN prefer global node-gyp@3.6.2 should be installed with -g

> node-sass@4.5.3 install /home/matthijs/Documents/skylines/ember/node_modules/node-sass
> node scripts/install.js

sh: 1: node: not found
skylines@0.0.0 /home/matthijs/Documents/skylines/ember
├─┬ ember-awesome-macros@0.38.4
│ ├─┬ ember-macro-helpers@0.16.2
│ │ └─┬ ember-weakmap@2.0.0
│ │   └─┬ ember-cli-babel@5.2.4
│ │     ├─┬ broccoli-babel-transpiler@5.7.2
│ │     │ └── broccoli-merge-trees@1.2.4 
│ │     └── ember-cli-version-checker@1.3.1 
│ └─┬ ember-string-ishtmlsafe-polyfill@2.0.0
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-bootstrap@1.0.0-rc.2 (git://github.com/kaliber5/ember-bootstrap.git#ca032c991a4fd1374a3044bcdb140c0d554810ab)
│ ├─┬ broccoli-stew@1.5.0
│ │ └── broccoli-merge-trees@1.2.4 
│ ├─┬ ember-assign-polyfill@2.0.2
│ │ └── ember-cli-version-checker@1.3.1 
│ ├─┬ ember-runtime-enumerable-includes-polyfill@2.0.0
│ │ └── ember-cli-version-checker@1.3.1 
│ └─┬ ember-wormhole@0.5.2
│   └─┬ ember-cli-htmlbars@1.3.4
│     └── ember-cli-version-checker@1.3.1 
├─┬ ember-cli@2.15.1
│ ├─┬ broccoli-babel-transpiler@6.1.2
│ │ └── broccoli-merge-trees@1.2.4 
│ ├─┬ ember-cli-legacy-blueprints@0.1.5
│ │ └── ember-cli-version-checker@1.3.1 
│ ├─┬ ember-cli-preprocess-registry@3.1.1
│ │ └── broccoli-merge-trees@1.2.4 
│ └─┬ ember-try@0.2.16
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-cli-chai@0.4.2
│ ├── broccoli-merge-trees@1.2.4 
│ └── ember-cli-version-checker@1.3.1 
├─┬ ember-cli-pace@0.1.0
│ └─┬ ember-cli-babel@5.2.4
│   ├─┬ broccoli-babel-transpiler@5.7.2
│   │ └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-cli-pretender@1.0.1
│ └── broccoli-merge-trees@1.2.4 
├─┬ ember-cli-shims@1.1.0
│ └── ember-cli-version-checker@1.3.1 
├─┬ ember-concurrency@0.8.10
│ └─┬ ember-getowner-polyfill@2.0.1
│   ├── ember-cli-version-checker@1.3.1 
│   └─┬ ember-factory-for-polyfill@1.2.0
│     └── ember-cli-version-checker@1.3.1 
├─┬ ember-cookies@0.0.13
│ └─┬ ember-cli-babel@5.2.4
│   ├─┬ broccoli-babel-transpiler@5.7.2
│   │ └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-cp-validations@3.4.0
│ ├─┬ ember-require-module@0.1.2
│ │ └─┬ ember-cli-babel@5.2.4
│ │   ├─┬ broccoli-babel-transpiler@5.7.2
│ │   │ └── broccoli-merge-trees@1.2.4 
│ │   └── ember-cli-version-checker@1.3.1 
│ ├─┬ ember-string-ishtmlsafe-polyfill@1.1.0
│ │ ├─┬ ember-cli-babel@5.2.4
│ │ │ └─┬ broccoli-babel-transpiler@5.7.2
│ │ │   └── broccoli-merge-trees@1.2.4 
│ │ └── ember-cli-version-checker@1.3.1 
│ └─┬ ember-validators@1.0.3
│   └─┬ ember-cli-babel@5.2.4
│     ├─┬ broccoli-babel-transpiler@5.7.2
│     │ └── broccoli-merge-trees@1.2.4 
│     └── ember-cli-version-checker@1.3.1 
├─┬ ember-font-awesome@3.0.5
│ └─┬ ember-cli-babel@5.2.4
│   ├─┬ broccoli-babel-transpiler@5.7.2
│   │ └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-href-to@1.13.0
│ └─┬ ember-cli-babel@5.2.4
│   ├─┬ broccoli-babel-transpiler@5.7.2
│   │ └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-intl@2.31.1
│ ├─┬ ember-intl-format-cache@2.6.0
│ │ └── broccoli-merge-trees@1.2.4 
│ ├─┬ ember-intl-messageformat@3.2.0
│ │ ├── broccoli-merge-trees@1.2.4 
│ │ └─┬ ember-intl-messageformat-parser@2.6.0
│ │   └── broccoli-merge-trees@1.2.4 
│ └─┬ ember-intl-relativeformat@3.3.0
│   └── broccoli-merge-trees@1.2.4 
├─┬ ember-intl-cp-validations@3.0.1
│ └─┬ ember-cli-babel@5.2.4
│   ├─┬ broccoli-babel-transpiler@5.7.2
│   │ └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-maybe-import-regenerator@0.1.6
│ └── broccoli-merge-trees@1.2.4 
├─┬ ember-power-select@1.9.6
│ └─┬ ember-text-measurer@0.3.3
│   └─┬ ember-cli-babel@5.2.4
│     ├─┬ broccoli-babel-transpiler@5.7.2
│     │ └── broccoli-merge-trees@1.2.4 
│     └── ember-cli-version-checker@1.3.1 
├─┬ ember-simple-auth@1.4.0
│ ├─┬ ember-fetch@3.3.1
│ │ └─┬ broccoli-templater@1.0.0
│ │   └─┬ broccoli-filter@0.1.14
│ │     └── mkdirp@0.3.5 
│ └─┬ ember-getowner-polyfill@1.2.5
│   ├─┬ ember-cli-babel@5.2.4
│   │ └─┬ broccoli-babel-transpiler@5.7.2
│   │   └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
├─┬ ember-source@2.13.4
│ ├── broccoli-merge-trees@1.2.4 
│ └── ember-cli-version-checker@1.3.1 
├─┬ ember-svg-jar@0.11.1
│ └── broccoli-merge-trees@1.2.4 
├─┬ ember-truth-helpers@1.3.0
│ └─┬ ember-cli-babel@5.2.4
│   ├─┬ broccoli-babel-transpiler@5.7.2
│   │ └── broccoli-merge-trees@1.2.4 
│   └── ember-cli-version-checker@1.3.1 
└─┬ eslint-config-simplabs@0.4.0
  └── ember-cli-version-checker@1.3.1 

npm ERR! Linux 4.10.0-33-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! node-sass@4.5.3 install: `node scripts/install.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the node-sass@4.5.3 install script 'node scripts/install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-sass
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/matthijs/Documents/skylines/ember/npm-debug.log
```

**Comment by Turbo87** [comment]
You should probably update your node.js version. I'd recommend to use https://github.com/creationix/nvm to install and manage node.js runtimes.

Regarding `npm install`, I forgot to update the README. We're using https://yarnpkg.com/ instead of npm because it automatically creates and manages a lockfile for the dependencies and is in general much more intuitive to use.

### tl;dr

```
curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.4/install.sh | bash
curl -o- -L https://yarnpkg.com/install.sh | bash

nvm install 8
yarn
```

**Comment by GliderGeek** [comment]
unfortunately, this didn't completely do the trick. i get the following error:

```
$ yarn
yarn install v1.1.0
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error ember-cli-babel@6.1.0: The engine "node" is incompatible with this module. Expected version "^4.5 || 6.* || 7.*".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

```

**Comment by Turbo87** [comment]
hmm interesting... there are several solutions to this:

- use `yarn --ignore-engines`
- use Node.js 6 via nvm
- update the relevant dependencies that don't know yet that they're compatible with Node 8

the last one is the best solution in the long term, but for now I'd recommend to use one of the other two.

**Comment by GliderGeek** [comment]
Awesome, seems to be working.
still need to add an example flight and the map server and them i am ready to update the airspaces 💪

**Comment by GliderGeek** [comment]
i managed to add a flight and view it locally.

the airspace is still taken from skylines.aero though. i have the feeling that i need to run a local map server and connect the app with this server instead of skylines.aero, but i can't find out how.

i ran the following command to load in the airspaces (after deleting 404 urls):
` ./manage.py import airspace mapserver/airspace/airspace_list.txt mapserver/airspace/airspace_blacklist.txt`

**Comment by Turbo87** [comment]
unfortunately I'm not sure myself how that part works. does https://github.com/skylines-project/skylines/blob/master/uwsgi/mapserver.py give you any pointers on how to get this running?

**Comment by Turbo87** [comment]
Thanks, and sorry for the delay! 😞
